.eslintignore

```diff
@@ -4,3 +4,4 @@ test/fixtures
 build/
 docs/
 protos/
+env-tests-logging/
```

Comment:
Our templates automatically populate this file, if you need this new directory you can add it to `excludes` an an extra argument [here](https://github.com/googleapis/nodejs-logging/blob/master/synth.py#L35).

Reply:
updated